### DISCUSSION TITLE
Render a beautiful mp4 video of Flows on local instead  of server.

### DISCUSSION THREAD

**Discussion by 7maucdn** [mobile-dev-inc/Maestro#1847] [Ideas]
Maestro makes easy to record a video without needing to clean up your desktop, arrange your windows, or deal with screen recording software.

The final output video are very beautiful, but it's only support to render on Maestro's server. I hope Maestro support to render on local soon.

**Comment by Fishbowler** [comment]
Done!
`maestro record --local`
